andela/LIT-Kigali1
# # Fix travis's coveralls build

#### What does this PR do?
Fixes travis-ci's coverall

#### Description of Task to be completed?

#### How should this be manually tested?
Navigate to travis to see the build using the following ling: `https://travis-ci.org/andela/LIT-Kigali1/builds/501454645`

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
